cmd/init-repo/main.go

```diff
@@ -139,18 +131,7 @@ func main() {
 		glog.Fatalf("Failed to create source repo directory %s: %v", BaseRepoPath, err)
 	}
 
-	if err := golang.InstallDefaultGoVersion(); err != nil {
```

Comment:
this is gone?

Reply:
This is moved to `InstallGoVersions`: https://github.com/kubernetes/publishing-bot/pull/232/files#diff-ca3cd93ee51f0f2890610d4399786b2bR44-R48

InstallGoVersions() is called in construct():

https://github.com/kubernetes/publishing-bot/blob/b1513c6d5c7dbbec57fbc1c3f3e339c24e16aba7/cmd/publishing-bot/publisher.go#L196-L201

Reply:
Ack.